pkgs/development/compilers/go/1.4.nix

```diff
@@ -107,10 +107,12 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p "$out/bin"
     export GOROOT="$(pwd)/"
-    export GOBIN="$out/bin"
+    export GOBIN="$(pwd)/bin"
```

Comment:
I think so. Technically GOBIN can be removed completely then, but I like explicit over implicit.